client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/EagerNearRemoteCache.java

```diff
@@ -11,11 +11,12 @@
  * @param <K>
  * @param <V>
  */
-public class NearRemoteCache<K, V> extends RemoteCacheImpl<K, V> {
+@Deprecated
+public class EagerNearRemoteCache<K, V> extends RemoteCacheImpl<K, V> {
```

Comment:
Do we have to worry about people referencing this class?  I am guessing not, but if so we can't really change the name.